src/terra-dev-site/IssueForm/Helper.jsx

```diff
@@ -55,7 +55,7 @@ const propTypes = {
 };
 
 // Templates used for preview, errors, and forms.
-const disclaimerTemplate = `When logging issues to Terra UI, it is helpful to have steps to reproduce the issue using a reduced test case, or examples from <a href="https://engineering.cerner.com/terra-ui" target="_blank">https://engineering.cerner.com/terra-ui</a>.
+const disclaimerTemplate = `When logging issues to Terra, it is helpful to have steps to reproduce the issue using a reduced test case, or examples from <a href="https://engineering.cerner.com/terra-ui" target="_blank">https://engineering.cerner.com/terra-ui</a>.
```

Comment:
It might be outside of the scope of this change, but I think the issue form is obscured a little too much. It's not intuitive to find and I'd bet most people don't know it's there. Is the issue form something we want to advocate/bring attention to?

Reply:
🤷‍♂️